<launch>

  <node pkg="camera1394stereo" type="camera1394stereo_node" name="stereo_camera" output="screen" >
    <param name="video_mode" value="format7_3" />
    <param name="color_coding" value="raw16" />
    <param name="bayer_pattern" value="grbg" />
    <param name="bayer_method" value="Simple" />
    <param name="stereo_method" value="Interlaced" />
    <param name="camera_info_url_left" value="" />
    <param name="camera_info_url_right" value="" />
  </node>
  
  <node pkg="dynamic_reconfigure" type="reconfigure_gui" output="screen" />

</launch>
